charts/ocis/templates/collaboration/route.yaml

```diff
@@ -0,0 +1,42 @@
+{{ if .Values.features.appsIntegration.enabled }}
```

Comment:
this is always generated, should be conditional for Openstack use case and will also fix the CI linting issues when disable there

Reply:
see also https://github.com/owncloud/ocis-charts/pull/653/files#diff-e314283a1b7efce113f2261b5b4cd28e8900d98dfd48aa14d08fc60eb6d3403dR1